docs/DeveloperGuide.md

```diff
@@ -270,21 +270,19 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 **MSS**
 
-1. User chooses to find a flashcard. 
+1. User chooses to find a flashcard.
 2. User enters the keyword they would like to search for.
-3. Bagel finds the flashcard and shows a list of flashcards that contain that keyword. 
-
-Use case ends.
+3. Bagel finds the flashcard and shows a list of flashcards that contain that keyword.
+  Use case ends.
```

Comment:
If you don't use don't leave an empty line in between, `Use case ends.` will be shown as being on the same line as `3. Bagel...` on the website